src/App.js

```diff
@@ -41,15 +42,27 @@ function App() {
     setTodoList(newTodoList);
   };
   return (
-    <>
-      <header>
-        
-        <h1>Todo List</h1>
-        </header>
-        <AddTodoForm onAddTodo={addTodo}/>
-        {isLoading ? (<p>Loading...</p>) : (
-        <TodoList todoList={todoList} onRemoveTodo={removeTodo} />)}       
-    </>
+    <div>
+      <BrowserRouter>
+        <Routes>
+          <Route exact path="/" element={
+            <>
+          <header>
+              <h1>Todo List</h1>
+            </header>
+            <AddTodoForm onAddTodo={addTodo} />
+            {isLoading ? (<p>Loading...</p>) : (
+              <TodoList todoList={todoList} onRemoveTodo={removeTodo} />)}
+              </>
+          }/>
```

Comment:
I learned something new today!!! 🎖️ 🏅 
I couldn't get this working until I looked at your example. 💯